doc/build_apps/get_started.rst

```diff
@@ -0,0 +1,504 @@
+Quick Start
+===========
+
+Application Development using CCF Overview
+-------------------------------------------
+
+What is CCF: The `Confidential Consortium Framework
+(CCF) <https://ccf.dev/>`__ is an open-source framework for building a
+new category of secure, highly available, and performant applications
+that focus on multi-party compute and data.
+
+-  Read the `CCF
+   overview <https://microsoft.github.io/CCF/main/overview/index.html>`__
```

Comment:
These links are best done with Sphinx's syntax, to avoid hardcoding a branch and a hosting location, eg.

```
:doc:`overview/index`
```

or

```
:ref:`overview/index:Overview`
```

See https://docs.readthedocs.io/en/stable/guides/cross-referencing-with-sphinx.html#the-ref-role for details.

Reply:
Done